src/sentry/conf/server.py

```diff
@@ -1140,6 +1140,8 @@ def SOCIAL_AUTH_DEFAULT_USERNAME():
     "organizations:performance-issues-post-process-group": False,
     # Enable internal view for bannerless MEP view
     "organizations:performance-mep-bannerless-ui": False,
+    # Enable updated landing page widget designs
+    "organizations:organizations:new-widget-designs": False,
```

Comment:
```suggestion
    "organizations:performance-new-widget-designs": False,
```
I think you accidentally typed `organizations` twice here? I also added `performance` here for consistency with our other flags if you're cool with that! 😄

Reply:
ah yes! thank you!